homeassistant/components/renson/switch.py

```diff
@@ -0,0 +1,80 @@
+"""Breeze switch of the Renson ventilation unit."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from renson_endura_delta.field_enum import CURRENT_LEVEL_FIELD, DataType
+from renson_endura_delta.renson import Level, RensonVentilation
+
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RensonCoordinator
+from .const import DOMAIN
+from .entity import RensonEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RensonBreezeSwitch(RensonEntity, SwitchEntity):
+    """Provide the breeze switch."""
+
+    _attr_icon = "mdi:weather-dust"
+    _attr_device_class = SwitchDeviceClass.SWITCH
+    _attr_has_entity_name = True
+    _attr_translation_key = "breeze"
+
+    def __init__(
+        self,
+        api: RensonVentilation,
+        coordinator: RensonCoordinator,
+    ) -> None:
+        """Initialize class."""
+        super().__init__("breeze", api, coordinator)
+
+        self._attr_is_on = False
```

Comment:
You can move this outside of the constructor and remove the constructor completely